ush/cam/evs_refs_plots_config.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/ksh 
+
+# ================================================================================================
+# NCEP EMC PYTHON PLOTTING OF CAM VERIFICATION
+# 
+# CONTRIBUTORS:     Marcel Caron, marcel.caron@noaa.gov, NOAA/NWS/NCEP/EMC-VPPPGB
+#
+# PURPOSE:          Set up configurations to plot METPLUS output statistics
+#
+# DESCRIPTION:      This configuration file defines environment variables that are meant to be 
+#                   edited by the user and that are called by python scripts at runtime to define 
+#                   the plotting task, resulting in one or more graphics output files. 
+#
+# BEFORE YOU BEGIN: Make sure to set up verif_plotting if you haven't already.  To set up, 
+#                   clone the Github repository:
+#
+#                   $ git clone https://github.com/MarcelCaron-NOAA/verif_plotting 
```

Comment:
Remove this line `$ git clone https://github.com/MarcelCaron-NOAA/verif_plotting`